### ISSUE TITLE
Strategy modified with ho_to_raw_codemod.py generates fatal exception during backtesting - AttributeError: 'float' object has no attribute 'value'

### ISSUE THREAD

**Issue by pascal00d** [iterativv/NostalgiaForInfinity#46]
Strategy modified with ho_to_raw_codemod.py generates fatal exception during backtesting:

```
2021-08-01 18:12:14,964 - freqtrade.optimize.backtesting - INFO - Running backtesting for Strategy NostalgiaForInfinityNext_0108
2021-08-01 18:12:15,189 - freqtrade - ERROR - Fatal exception!
Traceback (most recent call last):
  File "/home/pascal/freqtrade/freqtrade/main.py", line 37, in main
    return_code = args['func'](args)
  File "/home/pascal/freqtrade/freqtrade/commands/optimize_commands.py", line 53, in start_backtesting
    backtesting.start()
  File "/home/pascal/freqtrade/freqtrade/optimize/backtesting.py", line 517, in start
    min_date, max_date = self.backtest_one_strategy(strat, data, timerange)
  File "/home/pascal/freqtrade/freqtrade/optimize/backtesting.py", line 475, in backtest_one_strategy
    preprocessed = self.strategy.ohlcvdata_to_dataframe(data)
  File "/home/pascal/freqtrade/freqtrade/strategy/interface.py", line 730, in ohlcvdata_to_dataframe
    return {pair: self.advise_indicators(pair_data.copy(), {'pair': pair})
  File "/home/pascal/freqtrade/freqtrade/strategy/interface.py", line 730, in <dictcomp>
    return {pair: self.advise_indicators(pair_data.copy(), {'pair': pair})
  File "/home/pascal/freqtrade/freqtrade/strategy/interface.py", line 747, in advise_indicators
    return self.populate_indicators(dataframe, metadata)
  File "/home/pascal/freqtrade/user_data/strategies/NostalgiaForInfinityNext_01082021_Raw.py", line 3465, in populate_indicators
    informative_1h = self.informative_1h_indicators(dataframe, metadata)
  File "/home/pascal/freqtrade/user_data/strategies/NostalgiaForInfinityNext_01082021_Raw.py", line 3194, in informative_1h_indicators
    informative_1h['safe_pump_24_10'] = self.safe_pump(informative_1h, 24, self.buy_pump_threshold_10_24.value, self.buy_pump_pull_threshold_10_24.value)
AttributeError: 'float' object has no attribute 'value'
```

If you replace this line :
`    buy_pump_threshold_10_24 = DecimalParameter(0.4, 1.0, default=0.42, space='buy', decimals=3, optimize=False, load=True)
`

By this line:
`    buy_pump_threshold_10_24 = 0.42`

So maybe you need to replace this line:
`        informative_1h['safe_pump_24_10'] = self.safe_pump(informative_1h, 24, self.buy_pump_threshold_10_24.value, self.buy_pump_pull_threshold_10_24.value)
`

By this line (without ".value") ?
`        informative_1h['safe_pump_24_10'] = self.safe_pump(informative_1h, 24, self.buy_pump_threshold_10_24, self.buy_pump_pull_threshold_10_24)
`

**Comment by eSeR1805** [comment]
@anasyusef Seems valid this. Pinging you for whenever you have time for this.

**Comment by anasyusef** [comment]
Perfect. Currently working on the fix.

**Comment by anasyusef** [comment]
Partially fixed by #51. The current limitation is that references to HO values inside `buy_protection_params` will not get replaced. This has to be done manually. All other references will get replaced

**Comment by anasyusef** [comment]
This issue can be now closed as the limitation no longer applies on the latest update
